spec/controllers/v0/sign_in_controller_spec.rb

```diff
@@ -2347,6 +2347,22 @@
     let(:client_id_value) { client_config.client_id }
     let!(:client_config) { create(:client_config, logout_redirect_uri:) }
     let(:logout_redirect_uri) { 'some-logout-redirect-uri' }
+    let(:access_token) { SignIn::AccessTokenJwtEncoder.new(access_token: access_token_object).perform }
```

Comment:
any reason for all the spec changes? I'm pretty sure the only changes you need:

- remove places where `let!(:user)` is happening in `logout` specs
- update `let(:oauth_session)` to `let(:oauth_session) { create(:oauth_session, user_verification:) }`
- in context `when successfully authenticated` -> add `let(:user_verification) { create(:user_verification) }`

- in context `and authenticated credential is login.gov` -> `let(:user_verification) { create(:logingov_user_verification) }`